python_docs_theme/static/pydoctheme.css

```diff
@@ -1,11 +1,20 @@
 @import url("default.css");
 
+* {
+    box-sizing: border-box;
+}
```

Comment:
This should not be necessary and may break existing layouts

Reply:
OK, I will remove this.